04.class/memo_app.js

```diff
@@ -46,16 +46,14 @@ export class MemosApp {
 
   list() {
     for (const uuid in this.baseData) {
-      console.log(this.baseData[uuid][0]);
+      const row = this.baseData[uuid].split("\n");
```

Comment:
```suggestion
      const rows = this.baseData[uuid].split("\n");
```

配列なので複数形の方がいいですね。
あと、rowは「表の行」を表すことが多いので、ここはlinesの方が向いてるかも？

Reply:
変数名をlinesに修正いたしました！